### PR TITLE
Portfolio mobile changes.

### PR DESCRIPTION
**Pull request by jdbs9514**
It´s made with:

- HTML
- CSS
- Figma
- Flexbox

### CODE REVIEW

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @jdbs9514  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⛔ 🔴 🟥 

You have submitted a project, that requires the following;
- [ ] please add a good pull request summary of what you have done
- [ ] default branch should be the main branch, you can check how to change that [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request)
- [ ] please use the correct [README](https://github.com/microverseinc/readme-template/blob/master/README.md) TEMPLATE.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @jdbs9514 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 
- doesn't have a descriptive short summary please try to add one that tells in more detail what you've done in you Pull Request 
image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+
+<section>
+    <div class="title">
+        <h1 class="about">About Myself</h1>
+    </div>
+    <div class="description-test">
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    </div>
+    <div class="lets-connect">
+        LET´S CONNECT
```
Comment (keneogu) [inline index.html:184]: - [ ] Let's connect is a title. Make use of `hx` tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+    <div>
+        <input type="button" name="boton1" value="Get my resume">
+    </div>
+</section>
+
+<!--Seventh part-->
+
+<section>
+    <div class="languajes">
+     <h2>Languajes
```
Comment (keneogu) [inline index.html:204]: - [ ] From the project specification, The technology categories ("Languages", "Frameworks" and "Skills") should be wrapped in a `ul` tag, and each of them in `li` tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+    </div>
+    <div class="two-list">
+        <ul class="works">
+            <li class="java"><img src="images/Ellipse1.png" alt="js" class="js">JavaScript</li>
+            <li class="ht"><img src="images/Ellipse2.png" alt="Html" class="ht">HTML</li>
+            <li class="cs"><img src="images/Ellipse3.png" alt="css" class="cs">CSS</li>
+        </ul>
+    </div>
+    <div class="frameworks">
+        Frameworks
```
Comment (keneogu) [inline index.html:216]: - [ ] From the project specification, The technology categories ("Languages", "Frameworks" and "Skills") should be wrapped in a `ul` tag, and each of them in `li` tag.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+        <i class="fa-solid fa-angle-right"></i>
+    </div>
+    <div>
+        <hr>
+    </div>
+    <div class="skills">
+        Skills
```
Comment (keneogu) [inline index.html:223]: - [ ] From the project specification, The technology categories ("Languages", "Frameworks" and "Skills") should be wrapped in a `ul` tag, and each of them in `li` tag.

**Review by keneogu — CHANGES_REQUESTED** [review]
Hi @jdbs9514, 

Good job so far!

- well structured codes

There are some issues that you still need to work on to go to the next project but you are almost there!


## Invalid Code Review Request  ♻️

The linters are not working, please fix this and submit for a review

- [ ] Your hamburger menu is outside the mobile version design
screen2

- [ ] Please add the background color and follow the Figma design
Your design
screen1.
figma design
screen1a



_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

**Review by dasileker — APPROVED** [review]
### **Hi @jdbs9514,** 

### **Congratulations! 🎉**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 : 
 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**